packages/scandipwa/src/route/ProductPage/ProductPage.component.js

```diff
@@ -56,7 +57,8 @@ export class ProductPage extends PureComponent {
         isInformationTabEmpty: PropTypes.bool.isRequired,
         isAttributesTabEmpty: PropTypes.bool.isRequired,
         selectedBundlePriceExclTax: PropTypes.number.isRequired,
-        selectedInitialBundlePrice: PropTypes.number.isRequired
+        selectedInitialBundlePrice: PropTypes.number.isRequired,
+        ProductTabsDefaultValue: PropTypes.number.isRequired
```

Comment:
Probably we should use camelCase for this prop, as everywhere else to keep our code consistent.